framework/src/modules/interoperability/constants.ts

```diff
@@ -43,6 +43,7 @@ export const MAX_MODULE_NAME_LENGTH = 32;
 export const MIN_CROSS_CHAIN_COMMAND_NAME_LENGTH = 1;
 export const MAX_CROSS_CHAIN_COMMAND_NAME_LENGTH = 32;
 export const CHAIN_ID_LENGTH = 4;
+export const MIN_RETURN_FEE_PER_BYTE_LSK = BigInt(1000);
```

Comment:
```suggestion
export const MIN_RETURN_FEE_PER_BYTE = BigInt(1000);
```
I think this is not in `LSK`

Reply:
The name `MIN_RETURN_FEE_PER_BYTE_LSK` is suggested in LIP-45
https://github.com/LiskHQ/lips/blob/bf6143bd5654609fc5cdbaaaf580302e3affb9ed/proposals/lip-0045.md#notation-and-constants

I can change to `MIN_RETURN_FEE_PER_BYTE` or `MIN_RETURN_FEE_PER_BYTE_BEDDOWS` to make the const more self-explanatory